ui/templates/ui/collection_list.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% block title %}OVS | Collection List{% endblock %}
+{% block pagetitle %}OVS: Collection List{% endblock %}
+
+{% block content %}
+
+<ul>
+  {% for collection in collection_list %}
+    <li><a href="{% url "collection-detail" collection.hexkey %}">{{ collection.title }}</a></li>
+  {% empty %}
+    <li><i>Please create a collection first</i></li>
+  {% endfor %}
+</ul>
+
+<hr>
+
+<h2>Create New Collection</h2>
```

Comment:
@giocalitri if you want to hold off on the form issue until the next PR for #118 that's fine with me.  Everything else works great.